lib/osv/repos.py

```diff
@@ -24,16 +25,50 @@
 RETRY_SLEEP_SECONDS = 5
 
 
+class GitRemoteCallback(pygit2.RemoteCallbacks):
```

Comment:
where is GitRemoteCallback used, i dont see any callsite changes after move.

Reply:
In worker.py and importer.py, by users of the osv lib.